src/com/amazon/ion/benchmark/schema/constraints/Contains.java

```diff
@@ -0,0 +1,37 @@
+package com.amazon.ion.benchmark.schema.constraints;
+
+import com.amazon.ion.IonList;
+import com.amazon.ion.IonValue;
+
+/**
+ * Process the constraint 'Contains' which indicates that the list or S-expression is expected to contain all of specified
+ * values, in no particular order.
+ */
+public class Contains implements ReparsedConstraint {
+    private final IonList expectedContainedValue;
+
+    /**
+     * Initializing the newly created Object.
+     * @param expectedContainedValue represents the value of constraint 'contains'.
+     */
+    public Contains(IonValue expectedContainedValue) {
+        this.expectedContainedValue = (IonList) expectedContainedValue;
+    }
+
+    /**
+     * Helping to access the private attribute expectedContainedValue.
+     * @return IonList which represents the value of constraint 'contains'.
+     */
+    public IonList getExpectedContainedValue() {
```

Comment:
This represents multiple values, right? In other words, it returns a list of the values that are all required by the `contains` constraint. If that's true, naming this method `getExpectedContainedValues` would probably be more accurate.

Reply:
> This represents multiple values, right? In other words, it returns a list of the values that are all required by the `contains` constraint. If that's true, naming this method `getExpectedContainedValues` would probably be more accurate.

Yes, `getExpectedContainedValues` is more accurate to represent a list of values.